index.html

```diff
@@ -0,0 +1,140 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>Katherine Kocen | Protfolio</title>
+<!-- Links to CSS Style Sheets and Fonts -->
+            <link rel="stylesheet" href="css/main.css">
+            <link rel="stylesheet" href="css/normalize.css">
+            <link href="https://fonts.googleapis.com/css2?family=Lora:wght@400;700&display=swap" rel="stylesheet">
+            <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+    </head>
+
+
+    <body>
+        <header>
+            <div class="top">
+                <div class="full-width">
+                    <nav>
+                        <ul>
+                            <li><a href="#home">Home</a></li>
+                            <li><a href="#portfolio">Portfolio</a></li>
+                            <li><a href="#about">About</a></li>
+                            <li><a href="#contact">Contact</a></li>
+                        </ul>
+                    </nav>
+                </div>    
+            </div>
+
+            <div class="cover">
+                <div class="full-width">
+                    <h1>Hi There! I'm Katherine Kocen.</h1>
+                    <h2>Front-end developer, designer, and problem solver!</h2>
+            
```

Comment:
_nit_ Remove these new lines between each section unless your code blocks become hugeee to increase professionalism if someone looks at this code someday (anyone who visits your website can download it to see how to typed it up)